src/panels/lovelace/editor/hui-edit-view.ts

```diff
@@ -106,11 +120,10 @@ export class HuiEditView extends hassLocalizeLitMixin(LitElement) {
   }
 
   protected render(): TemplateResult {
-    return html`
-      ${this.renderStyle()}
-      <paper-dialog with-backdrop>
-        <h2>${this.localize("ui.panel.lovelace.editor.edit_view.header")}</h2>
-        <paper-dialog-scrollable>
+    let content;
+    switch (this._curTab) {
```

Comment:
This would seem to me like they should be different web components, especially because these editors are only going to grow in complexity.